content/docs/command-reference/cache/dir.md

```diff
@@ -6,7 +6,7 @@ using `dvc config cache`), or shows the current configured value.
 ## Synopsis
 
 ```usage
-usage: dvc cache dir [-h] [--global | --system | --local] [-u] value
+usage: dvc cache dir [-h] [--global | --system | --repo | --local] [-u] [value]
```

Comment:
Maybe `--repo` -> `--project` (similar to `gc`'s `--projects`, but that has a different meaning though), but I like the repo more, tbh.

Reply:
Let me understand what the option means exactly so can opine... ⌛

Reply:
I vote for `--project`.

- Repo sounds like Git.
- We already use the term "project" more for this concept (in docs).

Reply:
BUT, it's not obvious (from the name) what the difference is between "repo/project" and "local". Isn't the repo/project local? It wasn't an issue before since there was no `--repo` flag but I wonder if we should also rename `--local` to `--private` or something.

Reply:
In things like `dvc import/get` we use `DVC or Git repository` and not `project`. The term `project` seems a bit synthetic.

Regarding `--private` we have https://github.com/iterative/dvc/issues/3670 , but not sure it will be done for 2.0

Reply:
> Regarding --private we have iterative/dvc#3670

Thanks for reminding me about that one, OK cool no need to worry about it here then.

Reply:
I don't think "project" is synthetic. Maybe just a bit generic, like anything you're working on can be a project. But that's why when we need to be very explicit we can say "DVC project".

Repo basically implies Git and DVC does feature --no-scm mode so there are non-repo DVC project. That' the problem I see with "repo"... Plus let's think about where that comes from: *data repository* OK, does it work for *data repository*? I think that term is already sort of established in the subfield of data mining.

That said we currently have this terminology problem in docs so it's not like I have a definitive answer. But I guess whatever we decide here I can apply throughout (later).

Cc @shcheklein

Reply:
\* *code repository* OK, does it work for *data repository*?

Reply:
p.s.

> In things like dvc import/get we use DVC or Git repository

That is correct because import/get don't work with --no-scm projects, only repos.

Reply:
Let it be --project, I'm ok with it. I'll adjust the code and docs soon.

Reply:
@dberenbaum What do you think about `--repo` vs `--project`?

Reply:
`--project` works for me. I don't have a strong opinion as long as we try to maintain consistency.

Reply:
https://github.com/iterative/dvc/pull/5399